### DISCUSSION TITLE
fixed column width by pixel

### DISCUSSION THREAD

**Discussion by codelover2k** [TanStack/table#3525] [Q&A]
hi there!

It is possible to stretch your table fully by just give your table-tag a width of 100%, but I am struggling to give a column a fixed width by pixel, something like width: 90px; and be sure that this will never change.
Hope someone can help out, thanks!

**Comment by GoranSoce** [comment]
Looks like it's not possible. I can't believe it. Sunk so much time to implement a table, and now I find out I can't set the fixed width of a column. What a useless peace of shit library. Fuck you @TanStack.

**Reply by tannerlinsley** [reply]
To answer your question: This is 100% possible and there are at least a few examples that show how to do this. In fact, even if you don't use the built-in sizing tools, you can always just put a fixed number in the column meta and manually apply it as a `style` in your table markup. Have you asked about this feature already in the past? Did you reach out for any assistance prior to firing off this comment?

To address your behavior: This should be common sense, but I guess you need a reminder to watch your language and show some class. Your own personal online reputation and brand is at stake on this website just the same as everyone else's, so I would advise you not to tarnish it over something so petty. Take a breather and try again by professionally asking for some assistance.

For posterity: It can be easy to forget that OSS is 100% free and that no one here is under any obligation to assist you. If you choose a to have bad attitude about this, you're very likely never get the help you actually need. Regardless, I'm always happy to give second chances to help people who later decide to have a better attitude and treat people with respect.

**Comment by SannyMatt** [comment]
@GoranSoce  omg, don't be rude. 
Everything is possible with that library. Just google it.

**Reply by GoranSoce** [reply]
You're a special kind of imbecil aren't you.
